src/components/molecules/AdminTableLegend.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+	<div class="section">
+		<strong>{{ $t("components.molecules.adminfooterlegend.title") }}</strong>
+		<div class="consent-icon">
+			<li v-for="icon in icons" :key="icon.icon">
+				<span v-if="icon.class">
+					<base-icon
+						source="material"
+						:icon="icon.icon"
+						:color="icon.color"
+						:class="icon.class"
+					/>
+					<base-icon
+						source="material"
+						:icon="icon.icon"
+						:color="icon.color"
+						:style="icon.style"
+					/>
+					{{ icon.i18n }}
```

Comment:
```suggestion
					{{ icon.label}}
```
I think the name `i18n` is confusing and meaningless. It is not telling us, where it is used and what the intention is. Just from the name I would guess that it should be some kind of translation method???
Instead I would name it somewhat like `icon.label` or `icon.text` to emphasis that it is text that is actually visible.